test/fixtures/dialects/snowflake/limit.yml

```diff
@@ -318,10 +318,11 @@ file:
       - keyword: by
       - column_reference:
           naked_identifier: c1
-      limit_clause:
+      offset_clause:
```

Comment:
This now implies that `offset 3 fetch 3` and `fetch 3 offset 3` have different semantics. Is that an issue?

Reply:
This seems fine - what seems weird to be is that the `fetch` is showing as a limit clause, not a fetch clause. Would you mind taking a look at that as part of this PR?

Reply:
According to the snowflake docs: https://docs.snowflake.com/en/sql-reference/constructs/limit

> Both LIMIT (PostgreSQL syntax) and FETCH (ANSI syntax) are supported, and produce the same result.